spec/fixtures/constituency_api/SW149RQ.xml

```diff
@@ -0,0 +1 @@
+<Constituencies/>
```

Comment:
It might be clearer if this fixture was called "no_results.xml" rather than named after a specific postcode.